extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -259,14 +260,66 @@ export class MigrationCutoverDialog {
 					fontSize: '12px'
 				}).component();
 
+				const _emptyTableImage = view.modelBuilder.image().withProps({
+
+					iconPath: IconPathHelper.emptyTable,
+					iconHeight: '100px',
+					iconWidth: '100px',
+					height: '100px',
+					width: '100px',
+					CSSStyles:{
+						'text-align':'center'
+					}
+
+				}).component();
+
+				const _emptyTableText = view.modelBuilder.text().withProps({
+					value: "No backup files to show",
+					CSSStyles:{
```

Comment:
be sure to localize this string 'No backup files to show' by adding it to a localized constant in strings.ts